test/web-platform-tests/to-run.yaml

```diff
@@ -201,6 +196,13 @@ system-color-compute.html:
   "Inherited system color keyword is observable on caret-color": [fail, computedStyleMap not implemented]
   "Inherited system color keyword is observable on fill": [fail, computedStyleMap not implemented]
   "Inherited system color keyword is observable on stroke": [fail, computedStyleMap not implemented]
+  "System color doesn't compute to itself on box-shadow": [fail, Need cssstyle fix]
```

Comment:
Why do these regress?

Reply:
I haven't looked into the details, but I suspect that this may have coincidentally matched a color conversion table that jsdom had locally before.
My personal opinion is that it's expected to fail at this point.

Reply:
Found the reason.
For example, in the case of box-shadow, the propertyList.js contains:
```
[
  "box-shadow",
  {
    "href": "https://drafts.csswg.org/css-borders-4/#propdef-box-shadow",
    "initial": "none",
    "inherited": "no",
    "computedValue": "see individual properties",
    "name": "box-shadow",
    "styleDeclaration": [
      "box-shadow",
      "boxShadow"
    ]
  }
],
```

"computedValue" is set to "see individual properties" and not to "a/the computed color"

See lib/jsdom/living/helpers/style-rules.js#161